target/linux/ath79/image/generic.mk

```diff
@@ -979,6 +979,21 @@ define Device/sitecom_wlr-7100
 endef
 TARGET_DEVICES += sitecom_wlr-7100
 
+define Device/sitecom_wlr-8100
+  ATH_SOC := qca9558
+  DEVICE_VENDOR := Sitecom
+  DEVICE_MODEL := WLR-8100
+  DEVICE_ALT0_MODEL := X8 AC1750
```

Comment:
You also need DEVICE_ALT0_VENDOR := Sitecom

Reply:
Thanks, fixed!